config/routes.rb

```diff
@@ -1,3 +1,35 @@
+# == Route Map
+#
+#                                Prefix Verb   URI Pattern                                                                              Controller#Action
+#                                  root GET    /                                                                                        restaurants#index
+#                           restaurants GET    /restaurants(.:format)                                                                   restaurants#index
+#                                       POST   /restaurants(.:format)                                                                   restaurants#create
+#                            restaurant GET    /restaurants/:id(.:format)                                                               restaurants#show
+#                                       PATCH  /restaurants/:id(.:format)                                                               restaurants#update
+#                                       PUT    /restaurants/:id(.:format)                                                               restaurants#update
+#                                       DELETE /restaurants/:id(.:format)                                                               restaurants#destroy
+#           rails_amazon_inbound_emails POST   /rails/action_mailbox/amazon/inbound_emails(.:format)                                    action_mailbox/ingresses/amazon/inbound_emails#create
```

Comment:
why do we need that?

Reply:
generated with annotate gem, I'll remove the unrelated ones